randomuser/stepzen.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "RandomUser",
+  "description": "Generates random users for testing purposes using the randomuser.me API.",
+  "categories": ["testing"]
```

Comment:
Would you please change categories to "Dev Tools" from "testing" ? This impacts how it's displayed in the CLI ui under generators.

Reply:
Will do!